### ISSUE TITLE
Symfony/Drupal: TypeError: Cannot assign stdClass to reference held by property of type array

### ISSUE THREAD

**Issue by sascha-meissner** [kint-php/kint#420]
Hello, i´m running a drupal 10.1 with underlying symfony 6 on PHP 8.1.25.

Periodically i am getting an error when kint´ing stuff: 

```
TypeError: Cannot assign stdClass to reference held by property Symfony\Component\HttpFoundation\Session\Flash\FlashBag::$flashes of type array in Kint\Parser\Parser->parseArray() (line 379 of /srv/www/htdocs/my-dev/vendor/kint-php/kint/src/Parser/Parser.php). 

```

I´m not sure if this is a Kint, PHP, Symfony or Drupal problem, anyway im posting this issue here FYI

**Comment by jnvsor** [comment]
This should have been fixed in a700653 - is your kint version up to date?

**Comment by sascha-meissner** [comment]
Hey @jnvsor ! My bad, i was using 4.2.x .. using 5.x it works! thank you very much
